tutorials/roofit/rf615_simulation_based_inference.py

```diff
@@ -0,0 +1,231 @@
+## \file
+## \ingroup tutorial_roofit
+## \notebook
+## Use Simulation Based Inference (SBI) in RooFit
+##
+## This tutorial shows how to use SBI in ROOT. As reference distribution we
+## choose a simple uniform distribution. The target distribution is chosen to
+## be gaussian with different mean values.
+## The classifier is trained to discriminate between the reference and target
+## distribution.
+## We see how the neural networks generalize to unknown mean values.
+##
+## We compare the approach of using the likelihood ratio trick to morphing
+##
+## \macro_code
+## \macro_output
+##
+## \date July 2024
+## \author Robin Syring
+
+
+import ROOT
+import numpy as np
+from sklearn.neural_network import MLPClassifier
+
+# The samples used for training the classifier in this tutorial / rescale for more accuracy
+n_samples = 1000
+
+
+# morphing as a baseline 
+def morphing(setting):
+
+
+    # define binning for morphing 
+    bin_mu_x = ROOT.RooBinning(4, 0., 4.)
+    grid = ROOT.RooMomentMorphFuncND.Grid2(bin_mu_x)
```

Comment:
```suggestion
    grid = ROOT.RooMomentMorphFuncND.Grid(ROOT.RooBinning(4, 0., 4.))
```
Can we do this in one line? And maybe use `Grid` instead of `Grid2`. I think Grid2 only exists for backwards compatibility, but it is identical.